gluten-core/src/main/scala/io/glutenproject/extension/ExpandFallbackPolicy.scala

```diff
@@ -243,18 +246,44 @@ case class ExpandFallbackPolicy(isAdaptiveContext: Boolean, originalPlan: SparkP
     }
   }
 
+  private def countFallbackForVanillaSparkPlan(plan: SparkPlan): Int = {
```

Comment:
Using word "Fallback" for vanilla spark may be misleading, as in Gluten it means going back to vanilla spark plan instead of using gluten plan.

Reply:
Can we put it into the calculation of fallback cost?

Reply:
There is a conflict, we do not know the fallback vanilla plan when we are at fallback cost. We should first do fallback then we can compare the fallback vanilla plan vs. gluten plan.